jupyter_events/validators.py

```diff
@@ -30,19 +32,23 @@
     EVENT_CORE_SCHEMA["$id"]: EVENT_CORE_SCHEMA,
 }
 
-METASCHEMA_RESOLVER = RefResolver(
-    base_uri=EVENT_METASCHEMA["$id"], referrer=EVENT_METASCHEMA, store=SCHEMA_STORE
+METASCHEMA_REGISTRY = Registry().with_resources(  # type:ignore
```

Comment:
(And similarly here, you could choose to do
```suggestion
resources = [Draft7.create_resource(each) for each in (EVENT_METASCHEMA, PROPERTY_METASCHEMA, EVENT_CORE_SCHEMA)]
registry = resources @ Registry()
```

if you like that better).